From a32114cac62695329a2ced7c0a9700a34acd344f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 31 Oct 2016 16:01:24 -0700
Subject: [PATCH] TOOLS-1595 separate imgapi's pkgsrc packages: the stock set +
 imgapi-specific (no functional change) Reviewed by: Josh Wilsdon
 <josh@wilsdon.ca> Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 targets.json.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index 40e9c35..7fcdc90 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -588,16 +588,16 @@ cat <<EOF
     "pkgsrc": [
         "coreutils-8.23nb2",
         "curl-7.47.1",
-        "dateutils-0.3.1nb1",
-        "grep-2.22",
         "gsed-4.2.2nb4",
         "patch-2.7.5",
-        "xz-5.2.2",
+        "sudo-1.8.15",
+
+        "dateutils-0.3.1nb1",
         "haproxy-1.6.2",
         "openssl-1.0.2j",
         "stud-0.3p53nb5",
-        "sudo-1.8.15",
-        "the_silver_searcher-0.31.0"
+        "the_silver_searcher-0.31.0",
+        "xz-5.2.2"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-imgapi.git"}
-- 
2.21.0

